### PR TITLE
Functional tests - Add new test to edit quantity in product page (Classic & Hummingbird)

### PR DESCRIPTION
**Pull request by nesrineabdmouleh**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Add new test to edit quantity in product page (Classic & Hummingbird)
| Type?             | refacto
| Category?         | TE
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | Ci and nightly are :green_apple: 
| UI Tests          | https://github.com/nesrineabdmouleh/testing_pr/actions/runs/8734924317
| Fixed issue or discussion?     | no
| Related PRs       | no
| Sponsor company   |

### CODE REVIEW

File: tests/UI/campaigns/functional/FO/hummingbird/09_productPage/02_productPage/02_changeQuantity.ts [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+
+    it('should check the cart notifications number', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'checkNotificationsNumber', baseContext);
+
+      const notificationsNumber = await productPage.getCartNotificationsNumber(page);
+      expect(notificationsNumber).to.equal(14);
+    });
+
+    // @todo : https://github.com/PrestaShop/hummingbird/pull/600
+    it.skip('should set \'-24\' in the quantity input', async function () {
```
Comment (Progi1984) [inline tests/UI/campaigns/functional/FO/hummingbird/09_productPage/02_productPage/02_changeQuantity.ts]:
```suggestion
    it('should set \'-24\' in the quantity input', async function () {
```

File: tests/UI/campaigns/functional/FO/hummingbird/09_productPage/02_productPage/02_changeQuantity.ts [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+      await testContext.addContextItem(this, 'testIdentifier', 'updateQuantityByInput2', baseContext);
+
```
Comment (Progi1984) [inline tests/UI/campaigns/functional/FO/hummingbird/09_productPage/02_productPage/02_changeQuantity.ts:137]:
```suggestion
this.skip()
```

**Review by Progi1984 — COMMENTED** [review]
A small comment.

**Review by SD1982 — APPROVED** [review]

**Review by SD1982 — DISMISSED** [review]

File: tests/UI/campaigns/functional/FO/hummingbird/09_productPage/02_productPage/02_changeQuantity.ts (hunk 1 above)
Comment (nesrineabdmouleh) [reply tests/UI/campaigns/functional/FO/hummingbird/09_productPage/02_productPage/02_changeQuantity.ts]: Done

File: tests/UI/campaigns/functional/FO/hummingbird/09_productPage/02_productPage/02_changeQuantity.ts (hunk 2 above)
Comment (nesrineabdmouleh) [reply tests/UI/campaigns/functional/FO/hummingbird/09_productPage/02_productPage/02_changeQuantity.ts:137]: Fixed

**Review by Progi1984 — APPROVED** [review]

**Review by SD1982 — APPROVED** [review]

**Pull request merged**